src/language-js/printer-estree.js

```diff
@@ -481,17 +486,23 @@ function printPathNoParens(path, options, print, args) {
       //   )
 
       const hasJSX = isJSXNode(n.right);
-      const firstGroupIndex = parts.findIndex((part) => part.type === "group");
+
+      const firstGroupIndex = parts.findIndex(
+        (part) => typeof part !== "string" && part.type === "group"
```

Comment:
```suggestion
      @ts-ignore
      const firstGroupIndex = parts.findIndex((part) => part.type === "group");
```

to back out the change & ignore the type error - ??